cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -194,8 +195,12 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run external license
             throw ProgramResult(1)
         }
 
-        if (scanResults.hasIssues) {
-            println("The scan result contains issues.")
+        val hasSevereIssues = scanResults.scanResults.any { (_, results) ->
+            results.any { it.summary.issues.any { issue -> issue.severity > Severity.HINT } }
+        }
+
+        if (hasSevereIssues) {
+            println("The scan result contains errors or warnings.")
```

Comment:
To further align with `EvaluatorCommand`, how about adding a final line to this function (separated from the `if`-block with a blank line) saying

            println("The scan result contains hints only.")